ui-grid/ui-grid.d.ts

```diff
@@ -88,11 +88,11 @@ declare module uiGrid {
     }
     export interface IGridOptions {
         aggregationCalcThrottle?: number;
-        appScopeProvider?: ng.IScope;
+        appScopeProvider?: ng.IScope | Object;
```

Comment:
Actually, it does not as the internals extend it to the scope. For example:

                appScopeProvider: {
                    onDoubleClick: () => {
                        $rootScope.$emit("setActiveTab", 'member');
                    }
                },